docs/openstack-cinder-fips-encryption.md

```diff
@@ -0,0 +1,152 @@
+# FIPS Enabled Cinder Storage (LUKS)
+
+> [!NOTE]
+>
+> Genestack ships with Barbican key manager enabled by default for Cinder and Nova services. No further configuration is needed.
+
+> [!CAUTION]
+>
+> LUKS encrypted volumes are currently only supported in iSCSI workloads.  Ceph RBD is needs additional testing.  NFS backed Cinder volumes are known not to work:
+>
+> https://review.opendev.org/c/openstack/cinder/+/597148
+>
+> https://review.opendev.org/c/openstack/cinder/+/749155
+>
+> https://bugs.launchpad.net/nova/+bug/1987311
+>
+> https://review.opendev.org/c/openstack/nova/+/854030
+
+To create a FIPS enabled Cinder front end to be consumed by clients the folllowing command is run:
+
+> [!NOTE]
```

Comment:
I think we can use an admonition for this https://squidfunk.github.io/mkdocs-material/reference/admonitions/

    ```suggestion
!!! note

    ...
```